client/src/app/auth/login/page.tsx

```diff
@@ -64,63 +72,76 @@ const Login = () => {
+            to <span className="text-blue-600">KeySafe.</span>
+          </p>
+          <p className="mt-6 text-center font-medium md:text-left">Sign in to your account below.</p>
+          <form className="flex flex-col items-stretch pt-3 md:pt-8" onSubmit={onHandleSubmit}>
+            <div className="flex flex-col pt-4">
+              <div className="relative flex overflow-hidden rounded-md border border-gray-500 transition focus-within:border-blue-600">
+                <input
+                  type="email"
+                  id="login-email"
+                  className={`w-full flex-shrink appearance-none ${theme === 'light' ? 'bg-[white] text-gray-700 ' : 'bg-[#12141D] text-white '} border-gray-500 py-2 px-4 text-base  placeholder-gray-400 focus:outline-none`}
+                  placeholder="Email"
+                  value={email}
+                  onChange={(e) => setEmail(e.target.value)}
+                />
+              </div>
+            </div>
+            <div className="mb-4 flex flex-col pt-4">
+              <div className="relative flex overflow-hidden rounded-md border border-gray-500 transition focus-within:border-blue-600">
+                <input
+                  type="password"
+                  id="login-password"
+                  className={`w-full flex-shrink appearance-none ${theme === 'light' ? 'bg-[white]  text-gray-700' : 'bg-[#12141D] text-white'} border-gray-500 py-2 px-4 text-base  placeholder-gray-400 focus:outline-none`}
+                  placeholder="Password"
+                  value={password}
+                  onChange={(e) => setPassword(e.target.value)}
+                />
+              </div>
+            </div>
+            <a href="#" className="mb-6 text-center text-sm font-medium text-gray-600 md:text-left">Forgot password?</a>
             <button
-              onClick={onHandleSubmit}
               disabled={isLoginLoading}
-              className="w-full bg-blue-400 border rounded m-2 h-12 p-2 shadow-sm"
+              type="submit"
+              className="rounded-lg bg-blue-600 px-4 py-2 text-center text-base font-semibold text-white shadow-md outline-none ring-blue-500 ring-offset-2 transition hover:bg-blue-700 focus:ring-2 md:w-32"
             >
-              Submit
+              Sign in
             </button>
-
-            {isLoginLoading ? <LoadingSpinner /> : null}
-
-            <div className="">{errorMessage}</div>
-          </div>
-          <div className="flex flex-col items-center mt-12">
-            <p className="">
-              Don&apos;t have an account?{" "}
-              <span>
-                <Link
-                  className="text-teal-600 hover:underline"
-                  href="/auth/register"
-                >
-                  Register
-                </Link>
-              </span>
+          </form>
+          <div className="py-12 text-center">
+            <p className="text-gray-600">
+              Don't have an account?
```

Comment:
react not support this:  `don't` use like this `don&apos;t` 
Reason it will fails the build. 
Fix this where ever we are using this '